src/database/Database.cpp

```diff
@@ -222,6 +222,11 @@ Database::applySchemaUpgrade(unsigned long vers)
         // add tracking table information
         mApp.getHerderPersistence().createQuorumTrackingTable(mSession);
         break;
+    case 11:
+        mSession << "DROP INDEX IF EXISTS bestofferindex;";
```

Comment:
why do we need this index? I thought previous benchmarks demonstrated that they were not needed